testing/trino-testing/src/main/java/io/trino/testing/sql/TestTable.java

```diff
@@ -130,9 +123,9 @@ public void close()
         sqlExecutor.execute("DROP TABLE " + name);
     }
 
+    @Deprecated
     public static String randomTableSuffix()
```

Comment:
Just to avoid misunderstandings, do we still keep this method only to avoid eventual compile issues with ongoing PRs which may still be dependent on `randomTableSuffix()` utility method?